src/zino/tasks/bgpstatemonitortask.py

```diff
@@ -158,7 +158,8 @@ async def _get_cisco_bgp_info(self) -> Optional[list[BaseBGPRow]]:
             return None
 
         for oid, result in cisco_bgp_info.items():
-            result["cbgpPeer2RemoteAddr"] = ip_address(oid)
+            _addr_type, _addr_len, addr = oid[0], oid[1], oid[2:]
+            result["cbgpPeer2RemoteAddr"] = ip_address(bytes(addr))
```

Comment:
Is it useful to assign oid[0] and oid[1]  to variables when they are never used? Could simplify with
```suggestion
            result["cbgpPeer2RemoteAddr"] = ip_address(bytes(oid[2:]))
```
Or
```suggestion
            addr = oid[2:]
            result["cbgpPeer2RemoteAddr"] = ip_address(bytes(addr))
```

Reply:
I've left them in there on purpose as an explanatory note for future poor souls who might wonder WTH this is picking the magical `[2:]` part of the OID as the address (the preceding underscore of variable names is just the Pythonic way of indicating that this is a throwaway variable)